src/main/java/hw3/HeaderMenu.java

```diff
@@ -0,0 +1,118 @@
+package hw3;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class HeaderMenu {
```

Comment:
Please follow the order of the fields, constructors and methods in the class according to Java Code Convention